components/brave_wallet_ui/page/async/wallet_page_async_handler.ts

```diff
@@ -194,7 +194,8 @@ handler.on(WalletPageActions.removeImportedAccount.getType(), async (
 
 handler.on(WalletPageActions.viewPrivateKey.getType(), async (store: Store, payload: ViewPrivateKeyPayloadType) => {
   const keyringService = getWalletPageApiProxy().keyringService
-  const result = await keyringService.getPrivateKeyForKeyringAccount(payload.address, payload.coin)
+  // TODO (JL): Pass password here.
```

Comment:
cc @josheleonard ^^

Reply:
Need this to get merged first: https://github.com/brave/brave-core/pull/14669

Reply:
now passing password to `keyringService.getPrivateKeyForKeyringAccount` calls

[fix: use api-proxy & password to manage key reveal](https://github.com/brave/brave-core/pull/14731/commits/6507da55b03b5725df886ec0d71fa1b16b21d3b3)